.blazar.yaml

```diff
@@ -0,0 +1,18 @@
+env:
+  SKIP_VERSION_OVERRIDE: true
+  MAVEN_ARGS: -Drat.skip=true
+
+before:
+  - description: "Change version to 1.11.2-hubspot-SNAPSHOT"
+    commands:
+      - sed -i 's~^  <version>1.11.2-hubspot-SNAPSHOT</version>$~  <version>1.11.2-hubspot-SNAPSHOT</version>~' pom.xml
+
+buildpack:
+  host: git.hubteam.com
+  organization: HubSpot
+  repository: Blazar-Buildpack-Java
+  branch: v2
```

Comment:
Nor the buildpack section

Reply:
Without the buildpack I see 
![image](https://user-images.githubusercontent.com/3606665/50699835-f9560580-1016-11e9-9764-d3155362c852.png)

Any ideas?

Reply:
You're right, removing the buildpack section was predicated upon removing `.hubspot-blazar-discovery.yaml`, but if we need one then we need the other